test/models/report_test.rb

```diff
@@ -3,7 +3,30 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  include Devise::Test::IntegrationHelpers
```

Comment:
```suggestion
  include Devise::Test::IntegrationHelpers

```

includeの下は1行空けることが多いかも

Reply:
改行を追加しました。